### PR TITLE
Issue 2688 offline screens

### PR DESCRIPTION
**Pull request by ozten**
Shows primary offline screen based on the new `state` variable.

### CODE REVIEW

File: resources/static/test/mocks/xhr.js [hunk 1]
```diff
@@ -230,6 +231,7 @@ BrowserID.Mocks.xhr = (function() {
       var responseKey = request.type + " " + request.url + " " + responseName,
           response = xhr.responses[responseKey],
           typeofResponse = typeof response;
+      //console.log('responseKey=' + responseKey + ' response=' + response + ' typeofResponse=' + typeofResponse);
```
Comment (seanmonstar) [inline resources/static/test/mocks/xhr.js]: remove this dead log

File: resources/static/dialog/js/modules/primary_offline.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    this.close("cancel_state");
+  }
+
+  var Module = bid.Modules.PageModule.extend({
+    start: function(options) {
+      options = options || {};
+      var self = this,
+          parts = options.email ? options.email.split('@') : ['', ''];
+      options.idpName = parts[1];
+      self.renderWait("primary_offline", options);
```
Comment (seanmonstar) [inline resources/static/dialog/js/modules/primary_offline.js]: Why use `renderWait` here?

Comment (ozten) [inline resources/static/dialog/js/modules/primary_offline.js]: Good question... I dunno. Researching...

**Comment by ozten** [comment]
Updated.

**Pull request merged**